auction/messages.go

```diff
@@ -184,12 +184,12 @@ type BidOutcomes struct {
 
 	// Outcomes is a list of bid outcomes, one for every placed bid
 	// in the auction.
-	Outcomes []BidderOutcome `codec:"outcomes"`
+	Outcomes []BidderOutcome `codec:"outcomes,allocbound=-"`
```

Comment:
is this setting some sort of default for allocbound?

Reply:
I think that "-" means that there is no allocation limitation on arrays.